examples/connectivity/plot_adhd_covariance.py

```diff
@@ -5,10 +5,9 @@
 This example shows how to extract signals from regions defined by an atlas,
 and to estimate a covariance matrix based on these signals.
 """
- 
+n_subjects = 10  # Number of subjects to consider for group-sparse covariance
```

Comment:
Is there any reason to restrict to 10 subjects ? Just not sure what was the original intention. I would just stick to `datasets.fetch_adhd()` (i.e. 40 subjects) and remove the n_subjects line which wasn't used at all as you spotted.